### DISCUSSION TITLE
debian release @ 32bit

### DISCUSSION THREAD

**Discussion by Balza84** [RPTools/maptool#3650] [General]
We are trying to run and share Maptool server on old Asus ATOM EEE PC, but it's a 32bit architecture.

Any plan or hope for a 32-bit (i386) version of the .deb release?

Thanks in advance.

**Comment by Azhrei** [comment]
A 32-bit Java no longer exists, so the answer is no.  We have a channel on our Discord server for discussing MT on non-x86 hardware.  One user there has documented how to run MT on a Raspberry Pi, but it's a 64-bit build.  @cwisniew may have more to add.
